spec/lib/msf/core/modules/metadata/search_spec.rb

```diff
@@ -1,43 +1,78 @@
-RSpec.shared_examples_for 'Msf::Module::Search' do
-  describe '#search_filter' do
+require 'spec_helper'
+
+RSpec.describe Msf::Modules::Metadata::Search do
```

Comment:
I've decided to port over the existing slow search tests and I've tried to keep as close to the original specification as I could, to ensure there was no unintentional regressions introduced. In a future PR I'll most likely expand/change these tests.